src/code/views/value-slider-view.tsx

```diff
@@ -240,21 +283,51 @@ export class SVGSliderView extends React.Component<SVGSliderViewProps, SVGSlider
   }
 
   private handleDrag = (e) => {
+    if (this.isButtonEvent(e)) {
+      return;
+    }
     this.handleNoop(e);
+    this.sliderMoved = true;
     const { onValueChange } = this.props;
     if (onValueChange == null) { return; }
 
     const value = this.position(e);
     if (value !== this.props.value) {
+      this.checkForSliderWiggle(value);
       return onValueChange(value);
     }
   }
 
   private handleJumpAndDrag = (e) => {
+    if (this.isButtonEvent(e)) {
+      return;
+    }
     this.handleDrag(e);
     return this.handleStart(e);
   }
 
+  // shows the buttons if the user moves the slider around a small range
+  private checkForSliderWiggle(value: number) {
+    if (!this.state.showButtons) {
+      const {wiggleList} = this;
+      const {length} = wiggleList;
+      wiggleList.push(value);
```

Comment:
I am not seeing where this list gets trimmed.

Reply:
It gets reset on the click event handler which is the last event called.